templates/shared/forms/_country.html

```diff
@@ -3,7 +3,10 @@
   <label for="country">{{ country or 'Country' }}:</label>
   {% endif %}
   <select
+    {% if required == "false" %}
+    {% else %}
```

Comment:
```suggestion
    {% if required == "true" %}
```

Reply:
that wouldn't work, because all of the other pages are `required` and that means we need to pass the `"true"` for every single form we have on the site, so that it doesn't break. This way, we don't pass any values, because default is "else" which means `required` only when we pass "false" it'll be required.

I'd agree it's not the best, but it allows us to create an exception and not update all the forms that are using`_country`